Source/Csla.Shared/ALC/ALCManager.cs

```diff
@@ -0,0 +1,112 @@
+﻿#if NET5_0_OR_GREATER
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Runtime.Loader;
+
+namespace Csla.ALC
```

Comment:
I believe this should be in the Csla.Reflection namespace? I realize it is wrapping a `System.Runtime` concept, but it _feels_ more reflection-like than anything?

Also, by convention I think it should be `Alc` instead of `ALC`.

Reply:
I have no complains) You can name it whatever you like. I'm not sure that it has something with reflection, active AssemblyLoadContext just provides "closest" context for types caching/searching. Contexts are linked together via "child" -> "parent" chain, up to main, "Default", application context (like legacy AppDomain). So if type was registred in "Default" application context, but was accessed in some "dynamic" context, it will be looked up to "Default" context on this contexts chain, something like that.

Reply:
We could use a `Csla.Runtime` namespace to match the Microsoft convention, I'm fine with that as well.